### PR TITLE
combat-trainer.lic: Adjust the hysteresis setting name to match the o…

### PR DESCRIPTION
**Pull request by Sarvatt**
…thers, and add it to base.yaml

### CODE REVIEW

**Comment by rpherbig** [comment]
Seems reasonable. And the feature is young enough that I doubt very many people are using this setting. @Sarvatt can you post something on LNet a few times throughout the day?

FYI @b-Y-d

File: combat-trainer.lic [hunk 1]
```diff
@@ -22,7 +22,7 @@ class SetupProcess
     echo("  @cycle_armors: #{@cycle_armors}") if $debug_mode_ct
     @cycle_armors_time = settings.cycle_armors_time
     echo("  @cycle_armors_time: #{@cycle_armors_time}") if $debug_mode_ct
-    @armor_hysteresis = settings.armor_cycle_hysteresis
+    @armor_hysteresis = settings.cycle_armors_hysteresis
```
Comment (rpherbig) [inline combat-trainer.lic]: 💡 If you wanted to be extra safe, you could add a validator that `armor_cycle_hysteresis` is not set to true.

This feature is young enough that it's probably not necessary.

Comment (Sarvatt) [reply combat-trainer.lic]: I really think it's not necessary, only Byd is using it. I noticed today when I went to add after updating it that the name was a little askew.

Comment (Sarvatt) [reply combat-trainer.lic]: I did ask Byd what he thought about this change beforehand and he said he's all for it.

**Pull request merged**